### PR TITLE
feat(PE-6536): register all ants

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,39 +3,21 @@
...
-    "ts-node": "^10.9.1",
-    "typescript": "^5.1.6"
+    "prettier": "^3.3.2"
   },
-  "lint-staged": {
-    "**/*.{ts,js,mjs,cjs,md,json}": [
-      "prettier --write ."
-    ]
+  "dependencies": {
+    "@ar.io/sdk": "2.1.0-alpha.6"
```
Comment (dtfiedler) [inline package.json:21]: dev dep

File: tools/register-arns-ants.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+async function main() {
+  const arweave = Arweave.init();
+  const jwk = await arweave.wallets.generate();
+  const signer = createAoSigner(new ArweaveSigner(jwk));
+  const ario = IO.init();
+  const arnsRecords = await ario.getArNSRecords({
+    limit: 50_000,
+  });
+  const antIds = arnsRecords.items.map((record) => record.processId);
```
Comment (dtfiedler) [inline tools/register-arns-ants.js:14]: make this a set to avoid duplicates

File: tools/register-arns-ants.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    console.log(`Registering ${count++}/${antIds.length}...`);
+    await registry.send({
+      tags: [
+        { name: 'Action', value: 'Register' },
+        { name: 'Process-Id', value: antId },
+      ],
+      signer,
+    });
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+  }
```
Comment (dtfiedler) [inline tools/register-arns-ants.js:29]: batching here may make this more robust to timeouts - i.e. do 10 at a time and keep track of the last one you've submitted so you can start anew without having to spam more registers on retries

File: test/register.test.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+          { name: 'Action', value: 'Access-Control-List' },
+          { name: 'Address', value: STUB_ADDRESS },
+        ],
+      },
+      stateNoticeResult.Memory,
+    );
+
+    const affiliations = JSON.parse(allAntsResult.Messages[0].Data);
+    assert.strictEqual(affiliations.Owned[0], antId);
+  });
```
Comment (dtfiedler) [inline test/register.test.js:110]: this is a duplicate of the above and the `it` is inaccurate

Comment (dtfiedler) [inline test/register.test.js:110]:
```suggestion
```
this is a duplicate of the above and the `it` is inaccurate

File: tools/register-arns-ants.js (hunk 3 above)
Comment (atticusofsparta) [reply tools/register-arns-ants.js]: RE slack decided to stick with 1 by 1 for now

File: package.json (hunk 1 above)
Comment (dtfiedler) [reply package.json:21]: @atticusofsparta - this has not been moved

File: tools/register-arns-ants.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+  for (const antId of [...processIds]) {
+    console.log(`Registering ${count++}/${processIds.size}...`);
+    await registry.send({
+      tags: [
+        { name: 'Action', value: 'Register' },
+        { name: 'Process-Id', value: antId },
+      ],
+      signer,
+    });
```
Comment (dtfiedler) [inline tools/register-arns-ants.js:28]: nit - add a catch to avoid blowing up on a one-off failure (we already did it so not a big deal)

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**